### ISSUE TITLE
[MODULE] - Euclidean distance metric 

### ISSUE THREAD

**Issue by LeonardPuettmann** [code-kern-ai/bricks#230]
**Please describe the module you would like to add to bricks**
I would like to have a brick with which I can calculate the euclidean distance between two embedding vectors. 

**Do you already have an implementation?**
```python
from typing import Dict, any
import numpy 

def my_module(record: Dict[str, Any]) -> str:
    temp = vector_one - vector_two
    ed = np.dot(temp.T, temp)
    return ed
```

**Additional context**

**Comment by JWittmeyer** [comment]
not sure why but the default code cast the result to a string. I would assume no casting and using the datatype float will result in easier handling down the line

**Comment by LeonardPuettmannKern** [comment]
Yes that true. I'll have a look, thanks for bringing it up!

**Comment by LeonardPuettmannKern** [comment]
Implemented with #243
